packages/components/src/Banner/Banner.tsx

```diff
@@ -78,37 +79,38 @@ export function Banner({
     [styles.medium]: bannerWidth >= bannerWidths.medium,
   });
 
+  if (!showBanner) return null;
+
   return (
-    <>
-      {showBanner && (
-        <div
-          className={bannerClassNames}
-          ref={bannerRef}
-          role={type === "error" ? "alert" : "status"}
-        >
-          <div className={styles.bannerContent}>
-            {icon && <BannerIcon icon={icon} />}
-            <div className={styles.bannerChildren}>
-              <BannerChildren>{children}</BannerChildren>
-            </div>
-            {primaryAction && (
-              <div className={styles.bannerAction}>
-                <Button {...primaryAction} />
-              </div>
-            )}
-          </div>
-          {dismissible && (
-            <button
-              className={styles.closeButton}
-              onClick={handleClose}
-              aria-label="Close this notification"
-            >
-              <Icon name="cross" color={iconColors[type]} />
-            </button>
-          )}
+    <div
+      className={bannerClassNames}
+      ref={bannerRef}
+      role={type === "error" ? "alert" : "status"}
+    >
+      <div className={styles.bannerContent}>
+        {bannerIcon && <BannerIcon icon={bannerIcon} type={type} />}
+
+        <div className={styles.bannerChildren}>
+          <BannerChildren>{children}</BannerChildren>
         </div>
+
+        {primaryAction && (
+          <div className={styles.bannerAction}>
+            <Button {...primaryAction} />
+          </div>
+        )}
+      </div>
+
+      {dismissible && (
+        <button
+          className={styles.closeButton}
+          onClick={handleClose}
+          aria-label="Close this notification"
+        >
+          <Icon name="cross" color={iconColors[type]} />
```

Comment:
Icon should use `interactive--subtle` on all options now

Reply:
Post re-theme we could probably just drop in `ButtonDismiss` here

Reply:
only for retheme, yeah?

Reply:
For now, yeah